supervisor/shared/usb/usb.c

```diff
@@ -77,7 +74,7 @@ void usb_init(void) {
 }
 
 void usb_background(void) {
-    if (usb_enabled()) {
+    if (tusb_inited()) {
```

Comment:
If you're not calling `usb_enabled()` here, maybe we can get rid of that routine completely. Or call `usb_enabled()` here and it will call `tusb_inited()` itself.

Reply:
I will call usb_enabled() here, seem like it is use by common_hal_storage_remount() as well. So we need to keep it